Bio/SeqIO/PdbIO.py

```diff
@@ -17,6 +17,17 @@
 from .Interfaces import SequenceIterator
 
 
+def _res2aacode(residue, undef_code="X"):
+    """Return a residue's type as a one-letter code.
```

Comment:
I would change the docstring to: "Return the one-letter amino acid code from the residue name.". Type here is not really appropriate.